### PR TITLE
James/b/1657

### PR DESCRIPTION
**Pull request by James-Baloyi**
#1657

### CODE REVIEW

**Review by IvanIlyichev — CHANGES_REQUESTED** [review]
Hi @James-Baloyi. The code is not compilable. Please create pull requests only after successful local build and testing

**Comment by James-Baloyi** [comment]
Hi @IvanIlyichev, sorry it was a small oversight. I've fixed the unused imports

File: shesha-reactjs/src/designer-components/iconPicker/iconPickerWrapper.tsx [hunk 1]
```diff
@@ -26,27 +25,21 @@ interface IconPickerWrapperProps {
...
+    const { fontSize, color, readOnly, applicationContext, value, onChange, borderColor, borderRadius, borderWidth, backgroundColor, stylingBox, defaultValue } = props;
     const { data } = useFormData();
     const { globalState } = useGlobalState();
 
     const computedColor = useMemo(() => {
-        return customColor
-        ? executeScriptSync<string>(customColor, applicationContext)
-        : color;
-    }, [applicationContext, customColor, color]);
+        return color;
```
Comment (IvanIlyichev) [inline shesha-reactjs/src/designer-components/iconPicker/iconPickerWrapper.tsx]: Hi @James-Baloyi if this change is not a mistake - remove unneeded useMemo and use value directly

File: shesha-reactjs/src/designer-components/iconPicker/iconPickerWrapper.tsx [hunk 2]
```diff
@@ -26,27 +25,21 @@ interface IconPickerWrapperProps {
...
+    }, [applicationContext, color]);
 
     const computedBorderColor = useMemo(() => {
-        return customColor
-        ? executeScriptSync<string>(customColor, applicationContext)
-        : borderColor;
-    }, [applicationContext, customColor, borderColor]);
+        return borderColor;
```
Comment (IvanIlyichev) [inline shesha-reactjs/src/designer-components/iconPicker/iconPickerWrapper.tsx]: Hi @James-Baloyi if this change is not a mistake - remove unneeded useMemo and use value directly

File: shesha-reactjs/src/designer-components/iconPicker/iconPickerWrapper.tsx [hunk 3]
```diff
@@ -26,27 +25,21 @@ interface IconPickerWrapperProps {
...
+    }, [applicationContext, color, borderColor]);
 
     const computedIcon = useMemo(() => {
-        if (customIcon) return executeScriptSync<string>(customIcon, applicationContext);
-
-        return value;
-    }, [applicationContext, customIcon, value, borderColor]);
+        return defaultValue;
```
Comment (IvanIlyichev) [inline shesha-reactjs/src/designer-components/iconPicker/iconPickerWrapper.tsx]: Hi @James-Baloyi if this change is not a mistake - remove unneeded useMemo and use value directly

**Review by IvanIlyichev — CHANGES_REQUESTED** [review]
Hi @James-Baloyi. Check inline comments and make changes

**Comment by James-Baloyi** [comment]
@IvanIlyichev  the changes have been implemented

**Review by IvanIlyichev — APPROVED** [review]

**Pull request merged**